controllers/managedocs_controller.go

```diff
@@ -835,6 +836,10 @@ func (r *ManagedOCSReconciler) reconcileAlertmanagerConfig() error {
 		if smtpPassword == "" {
 			return fmt.Errorf("smtp secret does not contain a password entry")
 		}
+		smtpHTML, err := ioutil.ReadFile("templates/customernotification.html")
+		if err != nil {
+			return fmt.Errorf("unable to read customernotication.html file: %v", err)
```

Comment:
Typo in `unable to read customernotication.html file:`

Reply:
Done